frontend/packages/shipwright-plugin/integration-tests/testData/builds/shipwrightBuild.yaml

```diff
@@ -1,29 +1,31 @@
-apiVersion: shipwright.io/v1alpha1
+apiVersion: shipwright.io/v1beta1
```

Comment:
Please revert the integratin-tests changes. We need to wait for these changes until the operator is updated also on the OperatorHub.